src/redux/types/actions/colony.ts

```diff
@@ -66,13 +66,17 @@ export type ColonyActionTypes =
     >
   | UniqueActionType<
       ActionTypes.COLONY_MINT_TOKENS,
-      { colonyAddress: Address; amount: BigNumber },
+      {
+        colonyAddress: Address;
+        amount: BigNumber;
+        justification?: string;
```

Comment:
Any specific key that should be used here instead of `justification`?

Reply:
reason?